client/gutenberg/editor/style.scss

```diff
@@ -177,11 +177,9 @@ $gutenberg-theme-toggle: #11a0d2;
 	}
 
 	// UNSET CALYPSO DEFAULT STYLES
-	input[type='text'],
-	input[type='search'] {
```

Comment:
This line is still deleted but I think we should leave this whole block unchanged.

Reply:
Thanks for your patience with this, but I'm a bit lost. :sweat_smile:

I'm assuming that  /**` @format */" may not have worked as intended - I'm not familiar with it so perhaps you could please confirm if it has? Also, apologies for the ineptitude here, but I'm not exactly familiar with rebasing commits, and the GitHub support article suggests a requirement to have Write permissions to do so.

Reply:
Ah no worries, I can add `/** @format **/` in the follow up PR. I can also use the GitHub's Squash & Merge feature for that, but I'm not sure the lines will be attributed to you in that case.

Here is an explanation of how to do it: https://stackoverflow.com/questions/5189560/squash-my-last-x-commits-together-using-git

So basically, since this branch has 6 commits, you can `git rebase -i HEAD~6` and when the text editor opens replace `pick` with `f` in front of all but the first commit in the list. Hope that makes sense.

Reply:
Sorry if it's confusing, the above are all optional and I can work around them. The important thing is to verify that the code changes here indeed fix the https://github.com/Automattic/wp-calypso/issues/28457 issue (while at the same time not causing visual regressions that were previously reported in this PR).

Reply:
Provided everything works as intended and fixes the issue, it'd be great if you could work around them! Not to worry about any of the attribution of the lines - thanks for being so patient with this.

Reply:
Sorry for the delay @torres126, things were a bit busy after the release. :) Also thank you so much for sticking with this and trying to resolve it!

> Provided everything works as intended and fixes the issue, it'd be great if you could work around them! 

That doesn't seem to be the case at the moment, please see my comment here:
https://github.com/Automattic/wp-calypso/pull/28722#pullrequestreview-181463622

Reply:
No worries, thanks for the patience with this @vindl. I've made some changes and moved some CSS to the bottom, because it seemed to run into issues otherwise. However, I'm not too sure if there's going to be a problem with that, so please could you let me know if there is?

![sgdfsgfdsdfgsgfd](https://user-images.githubusercontent.com/43215253/49832088-919cf980-fd8d-11e8-9228-c05342a9b421.png)

Reply:
Unfortunately these rules would now be applied everywhere in Calypso, which would cause issues in unrelated places like `/settings/general/`:

<img width="746" alt="screenshot 2018-12-13 at 21 03 34" src="https://user-images.githubusercontent.com/1182160/49964549-0c474f80-ff1b-11e8-9114-e11a27266738.png">

These input boxes should take up the whole width of the parent container.

We need all our overrides wrapped in `.is-section-gutenberg-editor` I think.

Reply:
Noted! It seems like wrapping the override makes the password box play-up again though, but I'm still trying to figure out why this is the case.

Reply:
Added another commit which I think (hope!) solves issues from appearing in unrelated places, but I'm being fairly specific with `.edit-post-settings-sidebar__panel-block .components-panel__body input[type=text] ` since that's not wrapped.